src/main/java/uk/gov/hmcts/darts/cases/mapper/CasesMapper.java

```diff
@@ -93,7 +97,21 @@ public CourtCaseEntity addDefendantProsecutorDefenderJudge(CourtCaseEntity caseE
 
     @Transactional
     public SingleCase mapToSingleCase(CourtCaseEntity caseEntity) {
+        List<CaseRetentionEntity> caseRetentions = caseRetentionRepository.findAllByCourtCase(caseEntity);
         SingleCase singleCase = new SingleCase();
+
+        if (caseRetentions != null && !caseRetentions.isEmpty()) {
```

Comment:
No tests covering this logic? Surely that should be the first thing that goes in assuming doing TDD?

Reply:
RetentionPolicyTest.java added to cover new A/C updated in jira ticket